GymApp/GymApp/Migrations/20230612165918_test.cs

```diff
@@ -6,7 +6,7 @@
 namespace GymApp.Migrations
 {
     /// <inheritdoc />
-    public partial class newDb : Migration
+    public partial class test : Migration
```

Comment:
Dafuq, why are you renaming migrations? XD
This is a recipe for disaster, don't ever do that :P

Reply:
I did Get-Migration -Context GymApp.Data.AppDbContext
and got response
![image](https://github.com/MarcinKnyc/Gymnasium/assets/19415126/5aa35ce0-1625-453c-8452-137d8c2d7f6d)
Even tho the correct response would be
![image](https://github.com/MarcinKnyc/Gymnasium/assets/19415126/cd5abd67-6b98-4363-80bb-bec6490f0aff)